styles.css

```diff
@@ -0,0 +1,506 @@
+}
+
+:root {
+  --black: #1c1a19;
+  --wine: #730d15;
+  --red: #ff6b00;
+  --gray: gray;
+  --gray2: #262626;
+  --white: #f4f5f0;
+}
+
+body {
+  display: block;
+}
+
+* p {
+  word-spacing: 4px;
+}
+
+* .btn:hover {
+  color: var(--red);
+  border: 2px solid var(--red);
+  background-color: rgb(245, 216, 200);
+}
+
+* .btn:active {
+  background-color: rgb(200, 5, 25);
+  color: var(--white);
+}
+
+* .btn:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
+
+* a {
+  text-decoration: none;
+}
+
+/* This is form 0 px to 760 px */
+
+.navbar {
+  display: flex;
+  height: 48px;
+  background-color: var(--gray);
+  color: var(--white);
+}
+
+.logo {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  margin-left: 5%;
+}
+
+.logo-link {
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+#burguer-menu {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: right;
+  margin-right: 5%;
+}
+
+#logo-dat {
+  font-size: 5px;
+  padding: 0;
+  margin: 0;
+  color: var(--red);
+  height: 1rem;
+  display: flex;
+  align-items: flex-end;
+}
+
+header {
+  height: 682px;
+  background-color: var(--black);
+}
+
+.welcome-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 93%;
+  color: var(--white);
+  justify-content: center;
+}
+
+.welcome-section h1 {
+  font-size: 3.5rem;
+  line-height: 4rem;
+  text-align: left;
+  font-weight: 400;
+  word-spacing: 0.35px;
+  width: 85%;
+  color: var(--red);
+  margin-top: 120px;
+}
+
+.welcome-section h2 {
+  width: 85%;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  margin-top: 12px;
+}
+
+.welcome-section p {
+  font-size: 16px;
+  font-weight: 400;
+  width: 85%;
+  color: #fff;
+  line-height: 24px;
+  word-spacing: 1px;
+  z-index: 1;
+  font-style: normal;
+  margin-top: 12px;
+}
+
+.social-icons {
+  display: flex;
+  font-size: 20px;
+  width: 85%;
+  margin-top: 20px;
+  z-index: 1;
+  margin-bottom: 120px;
+}
+
+.social-icons li {
+  list-style: none;
+}
+
+.social-icons i {
+  margin-right: 15px;
+  margin-left: 5px;
+  color: #dbd8d7;
+  cursor: pointer;
+}
+
+.illustration-header-mobile {
+  display: flex;
+  left: 15%;
+  height: 240.89px;
+  width: 100%;
+  z-index: 0;
+  justify-content: flex-end;
+  align-items: flex-end;
+  max-height: 100%;
+}
+
+.vectors {
+  position: relative;
+  margin-top: -55%;
+  height: 256.89px;
+  width: 261.83px;
+}
+
+.works {
+  padding-top: 120px;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+}
+
+main hr {
+  width: 327px;
+  border: 1px solid #212121;
+  margin-top: 24px;
+  order: 1;
+}
+
+.daily-selection {
+  height: 568px;
+  margin-top: 80px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.img-placeholder {
+  background-color: var(--red);
+  width: 327px;
+  height: 237px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Kindly make sure that the image placeholder has the same margin and color as the Figma template.